js/main.js

```diff
@@ -323,3 +321,14 @@ function removeMovie(event) {
     }
   } emptyText();
 }
+
+window.addEventListener('load', spinner);
+
+function spinner(event) {
+  var $loading = document.querySelector('.loading');
+  $loading.className = 'loading hidden';
+}
+
+window.addEventListener('offline', function (event) {
+  alert('Sorry, there was an error connecting to the network! Please check your internet connection and try again.');
+});
```

Comment:
You should never use `alert`, it is un-styleable and stops all processes on the page. Instead, you should use a modal to convey this information.